openapi.yaml

```diff
@@ -1470,21 +1494,26 @@ paths:
         - name: indexType
           in: query
           description: Retrieve the OHLCV end of day by the index type.
-          required: true
+          required: false
           schema:
             type: string
             enum:
               - MWA
               - GWA
+              - LX
         - name: indexId
           in: query
           description: |
             Retrieve all the OHLCV values by the particular index id. The index id will differ based on the index type.
 
-            | Index Type | Index Id  |
-            | -----------|:---------:|
-            | MWA        |Market Id  |
-            | GWA        |Asset Id   |
+            | Index Type | Index Id  | Example Id | Description |
+            | -----------|:---------:|:----------:|             |
+            | MWA        |Market Id  |            |             |
+            | GWA        |Asset Id   |            |             |
+            | LX         |Index Id   | 191724d4-7915-491b-8b73-bc293e03a93e | [BLX](https://bravenewcoin.com/data-and-charts/indices/blx) |
+            | LX         |Index Id   | 6560acf3-1cb7-4fe4-aa19-fbd90c7b56ee | [ELX](https://bravenewcoin.com/data-and-charts/indices/elx) |
+            | LX         |Index Id   | ce36445a-2a33-4a0f-8b5b-e56175f8db60 | [XRPLX](https://bravenewcoin.com/data-and-charts/indices/blx) |
```

Comment:
Collapse together?

Reply:
Or another alternative is to have a section for LX indexes or just indexes in general that we reference instead of duplicating this Index section everywhere?